test/action_test.dart

```diff
@@ -26,62 +26,48 @@ void main() {
 
     setUp(() {
       action = Action<String>();
+      addTearDown(() async {
+        await action.dispose();
+      });
```

Comment:
#supernit. This could be reduced to:

```suggestion
      addTearDown(action.dispose);
```